cmd/proxy-server/main.go

```diff
@@ -72,6 +75,7 @@ var (
 	cfg   Config
 	// JWTSigningSecret is the secret string used to sign JWT tokens
 	JWTSigningSecret = "secret"
+	rdb              *redis.Client
```

Comment:
I think it would be better to use dependency injection in the `volumesHandler` rather than a package level variable.

Reply:
fixed, implemented dependency injection.